boards.txt

```diff
@@ -21,29 +21,36 @@ menu.maxqspi=Max QSPI
 menu.usbstack=USB Stack
 menu.debug=Debug
 
+# -----------------------------------
 # Adafruit Feather M0 (SAMD21)
-# ------------------------------
-adafruit_feather_m0.name=Adafruit Feather M0
+# -----------------------------------
+adafruit_feather_m0.name=Adafruit Feather M0 (SAMD21)
+
+# VID/PID for Bootloader, Arduino & CircuitPython
 adafruit_feather_m0.vid.0=0x239A
 adafruit_feather_m0.pid.0=0x800B
 adafruit_feather_m0.vid.1=0x239A
 adafruit_feather_m0.pid.1=0x000B
 adafruit_feather_m0.vid.2=0x239A
 adafruit_feather_m0.pid.2=0x0015
-adafruit_feather_m0.upload.tool=bossac18
+
+# Upload
+adafruit_feather_m0.upload.tool=bossac19
 adafruit_feather_m0.upload.protocol=sam-ba
 adafruit_feather_m0.upload.maximum_size=262144
 adafruit_feather_m0.upload.offset=0x2000
 adafruit_feather_m0.upload.use_1200bps_touch=true
 adafruit_feather_m0.upload.wait_for_upload_port=true
 adafruit_feather_m0.upload.native_usb=true
+
+# Build
 adafruit_feather_m0.build.mcu=cortex-m0plus
 adafruit_feather_m0.build.f_cpu=48000000L
 adafruit_feather_m0.build.usb_product="Feather M0"
 adafruit_feather_m0.build.usb_manufacturer="Adafruit"
 adafruit_feather_m0.build.board=SAMD_ZERO
 adafruit_feather_m0.build.core=arduino
-adafruit_feather_m0.build.extra_flags=-DARDUINO_SAMD_ZERO -DARM_MATH_CM0PLUS -DADAFRUIT_FEATHER_M0 -D__SAMD21G18A__ {build.usb_flags}
+adafruit_feather_m0.build.extra_flags=-D__SAMD21G18A__ -DADAFRUIT_FEATHER_M0 -DARDUINO_SAMD_ZERO -DARM_MATH_CM0PLUS {build.usb_flags}
```

Comment:
just re-arrange changes macros since we didn't do it in consistent way

Reply:
ok!